### PR TITLE
fix(clerk-sdk-node): Fix ESM build issues

### PR DESCRIPTION
**Pull request by nikosdouvlis**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.

### CODE REVIEW

**Comment by dimkl** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @dimkl - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/backend | 0.23.2-snapshot.81ca798 |
| @clerk/chrome-extension | 0.3.12-snapshot.81ca798 |
| @clerk/clerk-js | 4.48.0-snapshot.81ca798 |
| eslint-config-custom | 0.3.0 |
| @clerk/clerk-expo | 0.18.3-snapshot.81ca798 |
| @clerk/fastify | 0.5.1-snapshot.81ca798 |
| gatsby-plugin-clerk | 4.3.12-snapshot.81ca798 |
| @clerk/localizations | 1.19.0-snapshot.81ca798 |
| @clerk/nextjs | 4.21.4-snapshot.81ca798 |
| @clerk/clerk-react | 4.20.2-snapshot.81ca798 |
| @clerk/remix | 2.6.10-snapshot.81ca798 |
| @clerk/clerk-sdk-node | 4.10.8-snapshot.81ca798 |
| @clerk/shared | 0.19.1 |
| @clerk/themes | 1.7.5 |
| @clerk/types | 3.43.0-snapshot.81ca798 |

Tip: use the snippet copy button below to quickly install the required packages.
```sh
# @clerk/backend
npm i @clerk/backend@0.23.2-snapshot.81ca798
```
```sh
# @clerk/chrome-extension
npm i @clerk/chrome-extension@0.3.12-snapshot.81ca798
```
```sh
# @clerk/clerk-js
npm i @clerk/clerk-js@4.48.0-snapshot.81ca798
```
```sh
# eslint-config-custom
npm i eslint-config-custom@0.3.0
```
```sh
# @clerk/clerk-expo
npm i @clerk/clerk-expo@0.18.3-snapshot.81ca798
```
```sh
# @clerk/fastify
npm i @clerk/fastify@0.5.1-snapshot.81ca798
```
```sh
# gatsby-plugin-clerk
npm i gatsby-plugin-clerk@4.3.12-snapshot.81ca798
```
```sh
# @clerk/localizations
npm i @clerk/localizations@1.19.0-snapshot.81ca798
```
```sh
# @clerk/nextjs
npm i @clerk/nextjs@4.21.4-snapshot.81ca798
```
```sh
# @clerk/clerk-react
npm i @clerk/clerk-react@4.20.2-snapshot.81ca798
```
```sh
# @clerk/remix
npm i @clerk/remix@2.6.10-snapshot.81ca798
```
```sh
# @clerk/clerk-sdk-node
npm i @clerk/clerk-sdk-node@4.10.8-snapshot.81ca798
```
```sh
# @clerk/shared
npm i @clerk/shared@0.19.1
```
```sh
# @clerk/themes
npm i @clerk/themes@1.7.5
```
```sh
# @clerk/types
npm i @clerk/types@3.43.0-snapshot.81ca798
```

File: packages/sdk-node/tsup.config.ts [hunk 1]
```diff
@@ -10,12 +10,12 @@ export default defineConfig(overrideOptions => {
   const shouldPublish = !!overrideOptions.env?.publish;
 
   const common: Options = {
-    entry: ['./src/*.{ts,tsx,js,jsx}'],
-    bundle: false,
+    entry: ['src/index.ts', 'src/instance.ts'],
+    bundle: true,
```
Comment (anagstef) [inline packages/sdk-node/tsup.config.ts:14]: ❓ @nikosdouvlis why did we enable bundling here?

**Review by anagstef — APPROVED** [review]

**Review by dimkl — APPROVED** [review]

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**